### DISCUSSION TITLE
_DEEPIN_NO_TITLEBAR 如何隐藏标题栏？

### DISCUSSION THREAD

**Discussion by binex** [linuxdeepin/developer-center#3419] [Q&A | 问答板块]
在UOS上通过xprop设置_DEEPIN_NO_TITLEBAR属性，命令如下：`xprop -id xxxx -f _DEEPIN_NO_TITLEBAR 8c -set _DEEPIN_NO_TITLEBAR 1`，属性能设置上，但是标题栏还是显示了出来？看代码逻辑是不会绘制，但是为啥不生效呢？

**Comment by zccrs** [comment]
`xprop -root _NET_SUPPORTED|grep -i _DEEPIN_NO_TITLEBAR`

这样试一下能不能看到结果，如果不行，说明当前这个环境并不支持 _DEEPIN_NO_TITLEBAR
